### DISCUSSION TITLE
过期：播放完列表最后一个视频时，虽然停止播放，但pause属性为no

### DISCUSSION THREAD

**Discussion by zhongfly** [hooke007/mpv_PlayKit#14] [回收]
使用版本：
5.4.9.2版本，并覆盖使用ota-3补丁（未使用ota-02）
设置：（默认设置）
`loop-playlist=1`
复现过程：
1.播放某文件夹中最后一个视频
2.等待播放完毕或直接进度条拖到结尾
3.视频由于播放结束，所以停止播放，但osc中仍显示为正在播放，pause属性为no
![Snipaste_2021-08-18_18-18-50](https://user-images.githubusercontent.com/11155705/129882601-c60449ca-13ab-4e81-bab9-e8ecd7bf7900.png)
![Snipaste_2021-08-18_18-21-34](https://user-images.githubusercontent.com/11155705/129882605-73d0d917-5a76-4a52-8049-f439756fc6c5.png)
影响：
ontop_playback.lua脚本会在视频pause=yes时取消置顶，而此bug导致视频播放结束后仍然pause=no，导致视频虽然停止播放，但依然处于置顶

**Comment by zhongfly** [comment]
经测试，原版mpv.net 5.4.9.2有这个问题

**Comment by hooke007** [comment]
<s>这个。。。不是bug，`keep-open=yes` 的行为是播完暂停不是播完停止</s> (我眼花了重新看一遍

**Reply by zhongfly** [reply]
> 这个。。。不是bug，`keep-open=yes` 的行为是播完暂停不是播完停止

mpv不会这样，mpv.net才会，另外mpv.net里没搜到keep-open=yes这个设置

**Reply by zhongfly** [reply]
补充图片，mpv.net CM搜索不到keep-open（mpv.net的测试版可以），你忘记添加这选项了吧？
![Snipaste_2021-08-18_18-41-31](https://user-images.githubusercontent.com/11155705/129884636-db6b0016-b3aa-48e5-8e71-54c8f0d1e009.png)

**Reply by hooke007** [reply]
没忘，默认就是yes，没添加是防止初级用户误修改

**Comment by zhongfly** [comment]
我检查了mpv的设置也是`keep-open=yes`，但是最后暂停了，不是这个选项完全决定的
经过提醒，发现设置`keep-open-pause = yes`可以在播放结束时暂停

**Reply by hooke007** [reply]
。。。好吧，我没理解上游为什么会有这个默认设置
